R/build_stan_data.R

```diff
@@ -0,0 +1,197 @@
+#' Sets normal distribution parameters for sampling
+#'
+#' @param mean Mean of the normal distribution
+#' @param sd Standard deviation of the normal distribution
+#' @return List with specified statistics and name of the model
+#' @export
+sf_normal <- function(mean = 0, sd = 1) {
+  # Restricting normal inputs to be non-negative
+  if (mean < 0 || sd <= 0) {
+    msg <- paste0(
+      "Normal distribution here only accepts",
+      " non-negative values for mean and standard deviation"
+    )
+    message(msg)
+    stop()
```

Comment:
Can we pass the message directly as an error message?

```suggestion
    stop(
      "Normal distribution here only accepts",
      " non-negative values for mean and standard deviation"
    )
```